permission_handler_platform_interface/pubspec.yaml

```
@@ -3,20 +3,20 @@ description: A common platform interface for the permission_handler plugin.
 homepage: https://github.com/baseflow/flutter-permission-handler/tree/master/permission_handler_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.2
+version: 3.0.0-nullsafety
 
 dependencies:
   flutter:
     sdk: flutter
   meta: ^1.0.5
-  plugin_platform_interface: ^1.0.2
+  plugin_platform_interface: ^1.1.0-nullsafety
```

This can be updated to version 2.0.0:
```suggestion
  plugin_platform_interface: ^2.0.0
```